gateway/1.4/modules/ROOT/pages/flex-conn-tls-config.adoc

```diff
@@ -17,6 +17,9 @@ Before configuring the TLS context for a Flex Gateway Connected Mode, complete t
 
 * xref:flex-install.adoc[Install Flex Gateway]
 * xref:flex-{page-mode}-reg-run.adoc[Register and Run Flex Gateway]
+* Ensure you have the necessary permissions enabled:
+** Secrets Manager permissions: *Grant access to secrets*
```

Comment:
Does this  *Grant access to secrets* capitalization match what is in the UI? Same with *Deploy API Proxies* below?